presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -278,6 +278,23 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testColumnComment()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.test_column_comment (col1 bigint, col2 bigint, col3 bigint)");
+        execute("COMMENT ON COLUMN tpch.test_column_comment.col1 IS 'test comment'");
+        execute("COMMENT ON COLUMN tpch.test_column_comment.col2 IS ''");
+
+        System.out.println(computeScalar("SHOW CREATE TABLE test_column_comment"));
+
+        assertQuery(
+                "SELECT column_name, comment FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = 'test_column_comment'",
+                "VALUES ('col1', 'test comment'), ('col2', null), ('col3', null)");
```

Comment:
why comment for col2 is null and not `''`?

Reply:
In PostgreSQL, `COMMENT ON object IS ''` is the same as `COMMENT ON object IS NULL` (remove the existing comment). I couldn't find the document about the empty character, but even `pg_catalog.pg_description` table doesn't hold the empty comment. It means this is not driver's issue.

Reply:
Thanks, so i suggest: https://github.com/prestosql/presto/pull/1840#discussion_r340853598